### ISSUE TITLE
Connected in settings but not on dashboard

### ISSUE THREAD

**Issue by scfigg** [tuanha2000vn/hasskit#54]
I had this app working just fine, and I enabled multi-factor authentication. I uninstalled the app then reinstalled, and added it in with HTTPS and did the multi-factor authentication login, and it says it's connected. However when I go to the rooms or home page, it says I must connect to home assistant. Any suggestions?

**Comment by A320Peter** [comment]
I also use multi factor authentication with Authy but it works for me.

**Comment by tuanha2000vn** [comment]
Is the problem fixed, sometime unknow error happen and it clear by close the app and open it again.

**Comment by scfigg** [comment]
No, this is not fixed itself. I uninstalled the app, then reinstalled and nothing. I even have the latest update from the Google play store, and I will disconnect and reconnect the server again, and it shows it's connected... But the home page and room page just show that it needs to be connected to a server. Not sure what is going on. It was working for me since release, and before I enabled multi-factor authentication, I also switched my server to HTTP and the base URL because I was testing out my VPN connection. However I'm having problems with my VPN setup at the moment, so I went back to my old base URL and HTTPS which is always worked for me. I've been having problems since then. I've rebooted my home assistant box and everything works fine like it always has, other than this app. The official home assistant app also works for me
